templates/manage_certs.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Certificate management for haproxy load balancing server.
```

Comment:
@antoviaque Yes, I would like to write integration tests for this script, as I mentioned on the ticket.  I would be grateful if we could exclude this from the current task, though.